Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -227,31 +213,26 @@ private void StartProcessing(EntityUid toProcess, Entity<BiomassReclaimerCompone
 
         private bool CanGib(Entity<BiomassReclaimerComponent> reclaimer, EntityUid dragged)
         {
-            if (HasComp<ActiveBiomassReclaimerComponent>(reclaimer))
+            if (HasComp<ActiveBiomassReclaimerComponent>(reclaimer)
+                || !Transform(reclaimer).Anchored
+                || TryComp<ApcPowerReceiverComponent>(reclaimer, out var power) && !power.Powered)
                 return false;
 
             bool isPlant = HasComp<ProduceComponent>(dragged);
-            if (!isPlant && !HasComp<MobStateComponent>(dragged))
-                return false;
-
-            if (!Transform(reclaimer).Anchored)
-                return false;
-
-            if (TryComp<ApcPowerReceiverComponent>(reclaimer, out var power) && !power.Powered)
+            if (!isPlant
+                && !HasComp<MobStateComponent>(dragged)
+                || !isPlant
+                && reclaimer.Comp.SafetyEnabled
+                && !_mobState.IsDead(dragged))
```

Comment:
```suggestion
            if (!HasComp<ProduceComponent>(dragged) && (!HasComp<MobStateComponent>(dragged) || reclaimerComp.SafetyEnabled && !_mobState.IsDead(dragged)))
```
I'm not going to bother trying to format this properly on web (:pensive:), but you don't generally need to duplicate factors in an if, this also allows for much better short circuiting. Should have the same effect.